docs/docs/Components/components-tools.md

```diff
@@ -31,238 +31,237 @@ For an example, see [Make any component a tool](/agents-tool-calling-agent-compo
-## Glean Search API
+**Inputs**
 
-This component allows you to call the Glean Search API.
+| Name | Type | Description |
+|------|------|-------------|
+| metaphor_api_key | SecretString | API key for Exa Search (entered as a password) |
+| use_autoprompt | Boolean | Whether to use autoprompt feature (default: true) |
+| search_num_results | Integer | Number of results to return for search (default: 5) |
+| similar_num_results | Integer | Number of similar results to return (default: 5) |
 
-### Inputs
+**Outputs**
 
-| Name                   | Type         | Description                           |
-|------------------------|--------------|---------------------------------------|
-| glean_api_url          | String       | URL of the Glean API                 |
-| glean_access_token      | SecretString | Access token for Glean API authentication |
-| query                  | String       | Search query input                    |
-| page_size              | Integer      | Number of results per page (default: 10) |
-| request_options        | Dict         | Additional options for the API request (optional) |
+| Name | Type | Description |
+|------|------|-------------|
+| tools | List[Tool] | List of search tools provided by the toolkit |
 
-### Outputs
+</details>
 
-| Name    | Type      | Description                          |
-|---------|-----------|--------------------------------------|
-| results | List[Data]| List of search results               |
-| tool    | Tool      | Glean Search tool for use in LangChain|
+## Glean Search API
 
-## Google Search API
+This component allows you to call the Glean Search API.
 
-:::important
-This component is in **Legacy**, which means it is no longer in active development as of Langflow version 1.3.
-:::
+<details>
+<summary>Parameters</summary>
 
-This component allows you to call the Google Search API.
+**Inputs**
 
-### Inputs
+| Name | Type | Description |
+|------|------|-------------|
+| glean_api_url | String | URL of the Glean API |
+| glean_access_token | SecretString | Access token for Glean API authentication |
+| query | String | Search query input |
+| page_size | Integer | Number of results per page (default: 10) |
+| request_options | Dict | Additional options for the API request (optional) |
 
-| Name                   | Type         | Description                           |
-|------------------------|--------------|---------------------------------------|
-| google_api_key         | SecretString | Google API key for authentication     |
-| google_cse_id          | SecretString | Google Custom Search Engine ID        |
-| input_value            | String       | Search query input                    |
-| k                      | Integer      | Number of search results to return    |
+**Outputs**
 
-### Outputs
+| Name | Type | Description |
+|------|------|-------------|
+| results | List[Data] | List of search results |
+| tool | Tool | Glean Search tool for use in LangChain |
```

Comment:
Add periods and articles to the descriptions. Apply this throughout this PR. I won't comment on the rest that I find.